lib/linguist/generated.rb

```diff
@@ -795,7 +795,7 @@ def generated_sorbet_rbi?
       return false unless lines.count >= 5
       lines[0].match?(/^# typed:/) &&
       lines[2].include?("DO NOT EDIT MANUALLY") &&
-      lines[4].match?(/^# Please.*run.*`.*tapioca/)
+      lines[4].match?(/^# Please (run|instead update this file by running) `bin\/tapioca/)
```

Comment:
From the alert: 

"This regular expression that depends on a library input may run slow on strings starting with '# Pleaserun' and with many repetitions of 'run'."